frontend/server/src/main/java/com/amazonaws/ml/mms/http/ManagementRequestHandler.java

```diff
@@ -167,6 +168,11 @@ private void handleRegisterModel(ChannelHandlerContext ctx, QueryStringDecoder d
         int initialWorkers = NettyUtils.getIntParameter(decoder, "initial_workers", 0);
         boolean synchronous =
                 Boolean.parseBoolean(NettyUtils.getParameter(decoder, "synchronous", null));
+        int responseTimeout =
```

Comment:
This can be simplified as: 
        int responseTimeout = Integer.parseInt(NettyUtils.getParameter(decoder, "response_timeout", ConfigManager.getInstance().getDefaultResponseTimeout());